### PR TITLE
Updates for latest stuff.

### PR DESCRIPTION
**Pull request by geeknoid**
- No need to manually edit content in the generated sites since we'll be using the archive mode
of the sites.

- Generate an updated archives.yml that we now use to statically create the dropdown versioned
menu.

### CODE REVIEW

File: archive.istio.io/.idea/archive.istio.io.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ldemailly) [inline archive.istio.io/.idea/archive.istio.io.iml]: I don't think you want to commit this directory ?
  (what is it ? add it to .gitignore after you remove it from the PR)

File: archive.istio.io/build.sh [hunk 2]
```diff
@@ -40,11 +39,12 @@ do
   git checkout $TAG
   git pull 2> /dev/null
   echo "baseurl: /$NAME" > config_override.yml
-  for f in "${FILESTOPATCH[@]}"
+
+  echo "archives:" > _data/archives.yml
+  for rel in "${TOBUILD[@]}"
```
Comment (ldemailly) [inline archive.istio.io/build.sh]: I see from https://github.com/istio/istio.github.io/pull/844
you want that archives.yml to be part of site generation

let's go over the list 2x not n^2
ie create archives.yml once and copy it 
(you can add it to .gitignore)

**Comment by geeknoid** [comment]
Er, terrible PR.

All better now, PTAL.

File: archive.istio.io/build.sh [hunk 3]
```diff
@@ -40,17 +46,13 @@ do
   git checkout $TAG
   git pull 2> /dev/null
   echo "baseurl: /$NAME" > config_override.yml
-  for f in "${FILESTOPATCH[@]}"
-  do
-    mv  $f $f.orig 2> /dev/null && \
-    sed -e "s/>Istio/>Istio <span style='font-size: 0.6em;'>Archive $NAME<\/span>/" < $f.orig > $f && \
-    echo "*** Succesfully patched $f for $NAME"
-  done
+  cp ../archives.yml .
```
Comment (ldemailly) [inline archive.istio.io/build.sh]: isn’t it supposed to go in data/ ?

**Comment by ldemailly** [comment]
now #44 (for some reason)

**Pull request closed without merging**